### DISCUSSION TITLE
CUYANIMELIST || SESI 5 saat di connsol.log data tidak ada

### DISCUSSION THREAD

**Discussion by Destaanugra1** [deaafrizal/forum#50] [Troubleshooting Room]
![Screenshot (142)](https://github.com/deaaprizal/forum/assets/152362800/6cc50628-cd72-4561-9afc-9c63cfef2c4a)
![Screenshot (140)](https://github.com/deaaprizal/forum/assets/152362800/b8b6c014-dc2d-4776-9577-233b94b12757)
![Screenshot (141)](https://github.com/deaaprizal/forum/assets/152362800/26bd57cc-0402-491d-b349-2cb28796720d)
![Screenshot (139)](https://github.com/deaaprizal/forum/assets/152362800/0b26a02a-d53a-47c8-956b-c1d9b3e45baa)
kenapa yaa bg tolong padahal sudah sesuai semua

**Comment by deaafrizal** [comment]
coba yang di console log id nya dulu  tuh, dapet gak id dari url nya?

**Reply by Destaanugra1** [reply]
![Screenshot (144)](https://github.com/deaaprizal/forum/assets/152362800/529ed71d-3d7a-4968-85bd-242fce407fe3)
kekgii bg kayak gi,ama ya?

**Reply by andikaalakate** [reply]
Ada yang typo lagi itu, cuyy ... Di bagian url nya ...

**Reply by Destaanugra1** [reply]
yg di bagian mana bg

**Comment by lexaiko** [comment]
lu pake tema vscode apa bang

**Reply by yohanesokta** [reply]
high contrast mungkin 😄
